templates/helm-defaults.yml

```
@@ -5,7 +5,10 @@ generic-service:
   resources:
     requests:
       cpu: 128m
-      memory: 512Mi
+      memory: 1024Mi
+    limits:
+      cpu: 1024m
```

Not sure if it was intentional, but this is slightly more than 1 CPU (1024 millicores == 1.024 cores).  

If you were after a single core, then you can just set it to "1" (or 1000 millicores).
```suggestion
      cpu: "1" # or 1000m
```

This is currently set as the default here: https://github.com/ministryofjustice/cloud-platform-environments/blob/6d477a1f988ac2b318b1e3a92c9cf0b1ebb84fb1/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-probation-integration-services-prod/02-limitrange.yaml#L9